### PR TITLE
Add ability to propagate all supported pixelformats

### PR DESCRIPTION
**Pull request by arminn**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -30,6 +31,8 @@ message(STATUS "WITH_INPUT                    = ${WITH_INPUT}")
 message(STATUS)
 message(STATUS "WITH_MOCKBELIB                = ${WITH_MOCKBELIB}")
 message(STATUS)
+message(STATUS "WITH_MODIFIERS_HACK           = ${WITH_MODIFIERS_HACK}")
```
Comment (lorc) [inline CMakeLists.txt:34]: I would rename this to something more meaningful like `IGNORE_MODIFIER_VALUES`

File: src/displayBackend/wayland/WaylandZCopy.cpp [hunk 2]
```diff
@@ -379,10 +379,16 @@ void WaylandLinuxDmabuf::sOnModifiers(void *data,
 	 * We do not support passing modifiers from the frontend to the backend,
 	 * so ignore all formats with modifiers set.
 	 */
-	if (modifierHi == 0 && modifierLo == 0)
+	/*
+	 * If WITH_MODIFIERS_HACK is defined, we will skip modifiers check.
```
Comment (lorc) [inline src/displayBackend/wayland/WaylandZCopy.cpp:383]: We need to elaborate this comment: at least describe why it someone would need this enabled.

File: CMakeLists.txt [hunk 3]
```diff
@@ -15,6 +15,7 @@ OPTION(WITH_IVI_EXTENSION "build with wayland IVI Extension" ON)
 OPTION(WITH_INPUT "build with input backend" ON)
 OPTION(WITH_MOCKBELIB "build with mock backend lib" OFF)
 OPTION(WITH_DOC "build with documenation" OFF)
+OPTION(WITH_MODIFIERS_HACK "disable modifiers check" OFF)
```
Comment (lorc) [inline CMakeLists.txt:18]: "disable pixel format modifiers check (dangerous)"

**Review by lorc — COMMENTED** [review]
I think we can take this into our main tree, but only if we can clearly describe why this option is needed and what implications may be if it is enabled

**Comment by lorc** [comment]
`Reviewed-by: Volodymyr Babchuk <volodymyr_babchuk@epam.com>`

**Review by andr2000 — APPROVED** [review]
This is really DANGEROUS...

**Pull request merged**